src/main/java/org/example/racinggame/model/Car.java

```diff
@@ -0,0 +1,39 @@
+package org.example.racinggame.model;
+
+public class Car {
+
+    private static final int FORWARD_THRESHOLD = 4;
+    private static final int CAR_NAME_MAX_LENGTH = 5;
+    private final String name;
```

Comment:
상수와 인스턴스 변수를 구별하기쉽게 한줄 띄워주세요! 🙏

```suggestion
    private static final int CAR_NAME_MAX_LENGTH = 5;
    
    private final String name;
```